### PR TITLE
Revert "Improving docusaurus sidebar generation (#1927) (#14369)"

### PR DESCRIPTION
**Pull request by supertopher**
This reverts commit a2c194a11fdac0159e85cc77f5b6a54af8cd42bc.

this is a sad one.  The docs team found that renaming files was too cumbersome for their work stream.
me and @Amruta-Ranade felt this was the right path, but in use it was harder than using the unwieldy JSON.

OSS cares alot more about the order of specific documentation than cloud engineers do.

@Mykyta-Serbynevskyi this is a sad one.  we want to pull it before we get too far down the rabbit hole of the new directory structure though.  Thank you for your work.  Your work was great.  We may revisit this solution in the future but for now JSON works better for the docs team

TLDR the straw that broke he camel's back was renaming the second item in a folder with 14 items and then finding all the cross referenced links to those things when they renamed them

### CODE REVIEW

**Review by git-phu — APPROVED** [review]
😞

**Review by Amruta-Ranade — APPROVED** [review]
😭
